README.md

```diff
@@ -380,32 +385,57 @@ Run the demo using:
 resources/keyword_files/${SYSTEM}/porcupine_${SYSTEM}.ppn 0.5 ${AUDIO_DEVICE_INDEX}
 ```
 
+Replace `${LIBRARY_PATH}` with path to appropriate library available under [lib](/lib), `${SYSTEM}` with the
+name of the platform you are running on (`linux`, `raspberry-pi`, `mac`, `beaglebon`, and `jetson`), and `${AUDIO_DEVICE_INDEX}` with
+the index of your audio device.
+
 #### Windows
 
-**Requires MingW to run the demo.**
+List input audio devices with:
 
-At the root of the repository, build with:
+```console
+$ .\\demo\\c\\porcupine_demo_mic.exe --show_audio_devices
+```
+
+Run the demo using:
 
 ```console
-gcc -std=c99 -O3 -o demo/c/porcupine_demo_mic -I include/ demo/c/porcupine_demo_mic.c
+.\\demo\\c\\porcupine_demo_mic.exe lib/windows/amd64/libpv_porcupine.dll lib/common/porcupine_params.pv resources/keyword_files/windows/porcupine_windows.ppn 0.5 ${AUDIO_DEVICE_INDEX}
```

Comment:
there are both `\\` and `/` in this command. wondering if we can go with just one of them